collada_urdf/package.xml

```diff
@@ -25,6 +25,7 @@
   <build_depend>collada_parser</build_depend>
   <build_depend>liburdfdom-dev</build_depend>
   <build_depend>liburdfdom-headers-dev</build_depend>
+  <build_depend>pcrecpp</build_depend>
```

Comment:
I'm gonna go with no, based on the stuff I've looked at, I'm going to change this to `pcre` and `pcre-dev`. @isucan